pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -565,6 +565,25 @@ protected CompletableFuture<PartitionedTopicMetadata> internalGetPartitionedMeta
                 });
     }
 
+    protected CompletableFuture<Map<String, String>> internalGetPropertiesAsync(boolean authoritative) {
+        return validateTopicOperationAsync(topicName, TopicOperation.GET_METADATA)
+                .thenCompose(__ -> pulsar().getBrokerService().fetchPartitionedTopicMetadataAsync(topicName))
+                .thenCompose(metadata -> {
+                    if (metadata.partitions == 0 || topicName.isPartitioned()) {
```

Comment:
When `topicName.isPartitioned()` is true, the `metadata.partitions` may always equals to 0 here.